catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-converter/pom.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.codice.ddf.spatial</groupId>
+        <artifactId>wfs-v1.1.0</artifactId>
+        <version>2.12.0-SNAPSHOT</version>
+    </parent>
+    <packaging>bundle</packaging>
+    <artifactId>spatial-wfs-v1_1_0-converter</artifactId>
+    <name>DDF :: Spatial :: WFS :: 1.1.0 :: Converter</name>
+    <dependencies>
```

Comment:
I think you are missing the libs geospatial artifact

Reply:
https://github.com/codice/ddf/pull/2858/files#diff-4c4790d792825cd85b6f5da3a3e7d38eR148